### ISSUE TITLE
The SDK 50.2 AIR SDK when use HTMLLoader event.target.window alway will become 'null'.

### ISSUE THREAD

**Issue by snake0938** [airsdk/Adobe-Runtime-Support#2690]
I found that the 50.0.22 version of the AIR SDK  when use HTMLLoader to get web page  Complete Event of target.window   alway will become 'null',
, which makes it impossible to get the document of the webpage and the html code it contains. This problem is very disturbing, the same program It was normal when using adobe AIR 3.5 before.
![HtmlLoaderBug](https://github.com/airsdk/Adobe-Runtime-Support/assets/47775448/22558988-8425-4905-8408-0f25ce7fd280)

**Comment by ajwfrost** [comment]
Yes, the HTMLLoader functionality relied upon the webkit library which is no longer part of AIR. You could use AIR SDK 33.1.1.743 in order to keep that capability, but otherwise, applications will have to adapt to use the StageWebView "native" implementations (in 50.x the HTMLLoader class is a thin wrapper around this).

thanks

**Comment by xiaoChi888** [comment]
Any other way, will Harman plans to restore some functions of HTMLLoader? Although now the HTML content is not a part of Flash display-object List, some interaction between ActionScript and JavaScript are useful.
